server/notification-providers/freemobile.js

```diff
@@ -0,0 +1,24 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+
+class FreeMobile extends NotificationProvider {
+
+    name = "FreeMobile";
+
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully.";
+        try {
+            await axios.post(`https://smsapi.free-mobile.fr/sendmsg?msg=${encodeURI(msg.replace("🔴", "⛔️"))}`, {
```

Comment:
```suggestion
            await axios.post(`https://smsapi.free-mobile.fr/sendmsg?msg=${encodeURIComponent(msg.replace("🔴", "⛔️"))}`, {
```

Reply:
Usually, it should be `encodeURIComponent`, but I don't know how they handled this, you can test these characters.

![image](https://user-images.githubusercontent.com/1336778/193827385-19dd2497-f67e-4b42-a02b-bbbefdefaa25.png)

Reply:
I tested all these chars and all of them are working properly, should be good now